h5p-bildetema/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -8,16 +8,22 @@ export type LanguageSelectorProps = {
   currentLanguageCode: string;
   favLanguages: Language[];
   handleToggleFavoriteLanguage: (language: Language, favorite: boolean) => void;
+  isMobile: boolean | null;
 };
 
 export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
   languages,
   favLanguages,
   currentLanguageCode,
   handleToggleFavoriteLanguage,
+  isMobile,
 }) => {
-  const getAmountOfRows = (): number =>
-    Math.max(1, Math.ceil(languages ? languages.length / 2 : 0));
+  const getAmountOfRows = (): number => {
+    if (isMobile) {
+      return Math.max(1, Math.ceil(languages ? languages.length : 0));
```

Comment:
We don't need to round up if we don't do any division. 

```suggestion
      return Math.max(1, languages ? languages.length : 0);
```